etc/calls/testcall.py

```diff
@@ -1,19 +1,20 @@
-#!/usr/bin/python -u 
+#!/usr/bin/env -S python3 -u 
 
-import cPickle
+import pickle
```

Comment:
The version of the pyrolite library we use only supports the Pyro4 protocol, is that ok with that import change?

Reply:
Why import pickle and not _pickle which is the C based version, way faster than pickle?